### ISSUE TITLE
An update for the 'Installing a local instance for personal use' instructions needed

### ISSUE THREAD

**Issue by JoshuaCapel** [numbas/editor#419]
The instruction for installing a local instance for personal use are now out of date. These are the instructions at: http://numbas.github.io/editor/index.html

All version of the instructions refer to copying and modifying a file named database.py.dist, but such a file no longer exists in the project.

After working out how to search the log I found that commit d51690e36aa32201881f5d7d721bd0f9d7acd1c6 has removed it and added a script first_setup.py.

The steps up to copying and modifying the file numbas_editor/numbas/settings.py.dist were all mostly successful. Running python first_setup.py then seem to complete the setup (but I did need to create some folders that didn't exist).

------------------------------------------

Related to the instruction, a minor issue that did stop me during the setup was that I didn't have the package libjpeg8-dev installed. But I am not sure if this will be a common issue.

To clarify, I was following the instructions at http://numbas.github.io/editor/mac-local.html as I assumed they would be very similar to what I needed for a linux distribution (Ubuntu 18.04 LTS).

**Comment by JoshuaCapel** [comment]
Let me preface this by saying that I only have 15% of the skill set need to play around with Numbas, so feel free to ignore 85% of what I say if I don't see to understand what I am doing.

---------

Originally I was following the Ubuntu instructions, but they seemed to be putting the files in directories outside my home directory and setting up a server to be shared over the web. 

I only want a local version I can mess about/do developmental work on since I am almost certainly going to be using Numbas a lot in the coming year (and unlike other systems I must use, the backend does not need to be a closed-source mystery). The web-server set up was going to make it hard to edit the files without constantly becoming the root user.

I see the Mac and Windows instructions are labelled as 'recommended only for development', so I feel my choice was justified.

------------------------

By the way, I am not sure this issue is closed. The instruction at http://numbas.github.io/editor/mac-local.html seem to still reference the non-existent database.py.dist file. I did refresh my browser, so I am sure I am not just looking at a cached version.

**Comment by christianp** [comment]
You're right, I should have looked more closely. I've now put the personal/multi-user distinction back in, and added instructions to set up a personal instance on Ubuntu.
